src/CodeGeneration/ApiGenerator/Configuration/CodeConfiguration.cs

```diff
@@ -41,7 +42,15 @@ public static class CodeConfiguration
 			// these APIs are new and need to be mapped
 			"ml.set_upgrade_mode.json",
 			"ml.find_file_structure.json",
-			"monitoring.bulk.json"
+			"monitoring.bulk.json",
```

Comment:
Somewhat related to this, I think we should start letting these generate methods on the low level client and only use this list to block high level generation.

Reply:
Can follow up in a separate PR for this